velox/exec/MergeJoin.h

```diff
@@ -355,8 +386,25 @@ class MergeJoin : public Operator {
 
   std::optional<LeftJoinTracker> leftJoinTracker_{std::nullopt};
 
+  // Indices buffer used by the the left and right dictionary outputs. All
```

Comment:
> All
>   // output from left and right share the same dictionaries.

This is confusing to me. If left and right share dictionaries, then why do we need separate indices buffers?